Cargo.toml

```diff
@@ -736,5 +736,81 @@ field_names = "0.2.0"
 semver = "1.0.16"
 spinners = "4.1.0"
 include_dir = "0.7.3"
+bb8 = "0.8.5"
+bb8-postgres = "0.8.0"
+bigdecimal = "0.4.5"
+bytes-varint = "1"
+cargo_metadata = "0.15.4"
+cc = { version = "1", default-features = false }
+diffy = { version = "0.3", default-features = false }
+dotenvy = "0.15"
+downcast = "0.11.0"
+env_logger = "0.11.5"
+ethers = "2.0"
+futures-timer = "3.0.3"
+futures-util = "0.3.30"
+gcp-bigquery-client = "0.25.0"
+http-body-util = "0.1"
+jsonpath_lib = "0.3.0"
+k8s-openapi = "0.23.0"
+kube = "0.97.0"
+lazy_static = { version = "1", default-features = false }
+lexical-util = "1.0.6"
+libc = "0.2"
+log = "0.4.22"
+maplit = "1.0.2"
+memchr = "2"
+move-abstract-interpreter-latest = { path = "external-crates/move/crates/move-abstract-interpreter" }
+move-abstract-interpreter-v2 = { path = "external-crates/move/move-execution/v2/crates/move-abstract-interpreter" }
+move-bytecode-verifier-latest = { path = "external-crates/move/crates/move-bytecode-verifier" }
+move-bytecode-verifier-v0 = { path = "external-crates/move/move-execution/v0/crates/move-bytecode-verifier" }
+move-bytecode-verifier-v1 = { path = "external-crates/move/move-execution/v1/crates/move-bytecode-verifier" }
+move-bytecode-verifier-v2 = { path = "external-crates/move/move-execution/v2/crates/move-bytecode-verifier" }
```

Comment:
These versioned crates are intentionally not exposed as workspace dependencies so that access to them can be controlled via the `sui-execution` crate.